### PR TITLE
Add upgrade command for deployment pipeline

### PR DESCRIPTION
**Pull request by mrtuvn**
## Purpose of this pull request

This pull request (PR) add missing upgrade command for deployment pipeline

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/config-guide/deployment/pipeline/example/shared-configuration.html
https://devdocs.magento.com/guides/v2.4/config-guide/deployment/pipeline/example/cli.html
https://devdocs.magento.com/guides/v2.4/config-guide/deployment/pipeline/example/environment-variables.html

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/_includes/config/split-deploy/example_build-sync.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ To update your build system:
    git pull mconfig m2.2_deploy
    ```
 
+1. Run Upgrade Magento:
```
Comment (BarnyShergold) [inline src/_includes/config/split-deploy/example_build-sync.md:13]:
```suggestion
1. Run upgrade Magento:
```

**Review by BarnyShergold — APPROVED** [review]
It's so small I'll approve now.

File: src/_includes/config/split-deploy/example_build-sync.md (hunk 1 above)
Comment (mrtuvn) [reply src/_includes/config/split-deploy/example_build-sync.md:13]: Updated as you suggest

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Comment by hostep** [comment]
I don't understand how this got approved. This is incorrect information I believe.

Correct me if I'm wrong, but during step 3 on these pages, Magento doesn't have access yet to the database, so running `setup:upgrade` will crash at that point. No?

**Comment by hostep** [comment]
If I understand it correctly, this PR was probably created because of the discussion in https://github.com/magento/magento2/issues/32149 and https://github.com/magento/devdocs/issues/8690

**Pull request merged**